src/schema/core.yaml

```diff
@@ -1324,26 +1393,18 @@ classes:
     is_a: OntologyClass
     description: >-
       An atom or molecule that can be represented with a chemical formula. Include lipids, glycans, natural products, drugs.
-      There may be different terms for distinct acid-base forms, protonation states
+      There may be different terms for distinct acid-base forms, protonation states. A chemical entity is a 
+      physical entity that pertains to chemistry or biochemistry.
     comments:
       - As with the parent OntologyClass, we will not assign an nmdc id pattern or typecode to this class.
     slots:
       - chemical_formula
-      - inchi
-      - inchi_key
-      - smiles
-
     see_also:
       - https://bioconductor.org/packages/devel/data/annotation/vignettes/metaboliteIDmapping/inst/doc/metaboliteIDmapping.html
     id_prefixes:
-      - cas
       - CHEBI
-      - CHEMBL.COMPOUND
-      - DRUGBANK
+      - MS
       - HMDB
```

Comment:
I propose to remove HMDB unless we have a use case within scope.

Reply:
agree; done.